airflow-core/tests/unit/api_fastapi/core_api/routes/public/test_dags.py

```diff
@@ -58,6 +63,7 @@ class TestDagEndpoint:
 
     @staticmethod
     def _clear_db():
+        clear_db_connections()
```

Comment:
Same qn here

Reply:
See above - this one is **clearly** good to have and simply missing. Those thests rely on test git connection not present in the DB - so it's a good practice to clear the connection table to avoid accidental side-effects from other tests - we clear other tables in the next line, so clearing the connection tablee **should** be added here.